src/MOI_wrapper.jl

```diff
@@ -432,16 +432,23 @@ end
     end
     # qmatind is ::Vector{Cint}, so we convert back to ::Vector{Int}.
     return sparse(Int.(qmatind), qmatcol, qmatval)
- end
+end
+
+const integer_types = Set{Symbol}([:MILP, :MIQP, :MIQCP])
+const continuous_types = Set{Symbol}([:LP, :QP, :QCP])
 
 function LQOI.make_problem_type_integer(optimizer::Optimizer)
     optimizer.inner.has_int = true
-    set_prob_type!(optimizer.inner, :MILP)
+    prob_type = get_prob_type(optimizer.inner)
+    prob_type in integer_types && return
+    set_prob_type!(optimizer.inner, prob_type_toggle_map[prob_type])
```

Comment:
Add a comment that `prob_type_toggle_map` is defined in CplexSolverInterface.jl.